{
    "namespace": "django.core.handlers.exception"
}